release_docs/RELEASE.txt

```diff
@@ -232,41 +234,29 @@ New Features
       of the HDF5 install directory.
 
 
-Support for new platforms, languages and compilers
+Support for New Platforms, Languages and Compilers
 ==================================================
     -
 
 
-Bug Fixes since HDF5-1.12.2 release
+Bug Fixes Since HDF5-1.12.2 Release
 ===================================
     Library
     -------
-    - Fixed CVE-2018-11202
+    - Fixed CVE-2018-11202, a malformed file could result in chunk index
+      memory leaks.
 
-      A malformed file could result in chunk index memory leaks. Under most
-      conditions (i.e., when the --enable-using-memchecker option is NOT
-      used), this would result in a small memory leak and and infinite loop
-      and abort when shutting down the library. The infinite loop would be
-      due to the "free list" package not being able to clear its resources
-      so the library couldn't shut down. When the "using a memory checker"
-      option is used, the free lists are disabled so there is just a memory
-      leak with no abort on library shutdown.
+      Under most conditions (i.e., when the --enable-using-memchecker option is
+      NOT used), this would result in a small memory leak and and infinite loop
+      and abort when shutting down the library. The infinite loop would be due
+      to the "free list" package not being able to clear its resources so the
+      library couldn't shut down. When the "using a memory checker" option is
+      used, the free lists are disabled so there is just a memory leak with no
+      abort on library shutdown.
```

Comment:
Does the last sentence describe how things work after a change to fix the problem?  If so, would starting the sentence with "Now" , or inserting "now" in "the free lists are now disabled" make this clearer?